Framework/Kernel/inc/MantidKernel/Material.h

```diff
@@ -196,6 +199,8 @@ class MANTID_KERNEL_DLL Material final {
   void saveNexus(::NeXus::File *file, const std::string &group) const;
   void loadNexus(::NeXus::File *file, const std::string &group);
 
+  bool hasValidxRayAttenuationOverride();
```

Comment:
Similar comment - I suggest you call this hasValidxRayAttenuationProfile() rather than using the word override